product_redirection_to_front_office/__openerp__.py

```diff
@@ -0,0 +1,50 @@
+# -*- encoding: utf-8 -*-
+#
+# OpenERP, Open Source Management Solution
+#    This module copyright (C) 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+{
+    'name': 'Product Redirection to Front Office',
+    'version': '1.0',
+    'author': 'Savoir-faire Linux',
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'category': 'Sales',
+    'summary': ('In product.template, add a button to see the product '
+                'into the shop.'),
+    'description': """
+Product Redirection to Front Office
```

Comment:
Extract this into a file called README.rst and remove it from here for Github and Odoo compatibility.